### DISCUSSION TITLE
Submit ideas for your presentation

### DISCUSSION THREAD

**Discussion by birkenkrahe** [birkenkrahe/ai482#4] [Ideas]
Submit your ideas here as a comment on this post by Wednesday, September 1, 9 AM. 

- Put the tentative title of your presentation topic (or topics, if there are several you're thinking about) as a headline at the top of your post. 
- In the body of your post, put references or 
- thoughts (e.g. why you want to present this topic - what motivates), or 
- concerns (which difficulties concern you).

References (literature, sites, blogs etc.) are most important if you have them, because they help to validate the topic.

Your topics should have available DATA, be DOABLE, you should already have some KNOWLEDGE (i.e. use what you already know or have learnt), and you should be MOTIVATED to research them.

We'll discuss this on Wednesday. See also [followup from August 30](https://github.com/birkenkrahe/ai482/blob/main/3_ai_projects/followup_08_30_2021.md).

**Comment by SRhoden42** [comment]
[AI and AVs Refrences.docx](https://github.com/birkenkrahe/ai482/files/7102931/AI.and.AVs.Refrences.docx)
## AI and Autonomous Driving Technology 

- I have been interested in EVs and Autonomous tech since the 9th grade and I plan to explore it further in my solo presentation.

- I feel as though this technology is going to be very prominent in the future and that it would be valuable to have research knowledge about it. ([D'Allegro](https://www.investopedia.com/articles/investing/052014/how-googles-selfdriving-car-will-change-everything.asp)) Also, I have had a long-term interest in robotics growing up with my very first project, 10 years ago, being a little 3 wheel robot that used a sensor to follow a line along a course, so this feels like a grown-up version of that which makes it interesting to explore. 

- I am concerned that I may have issues with finding good real-world test data to work with in regards to this project. 

### A few references:
D'Allegro J (20 June 2021) How Google's Self-Driving Car Will Change Everything. [Online: investopedia.com](https://www.investopedia.com/articles/investing/052014/how-googles-selfdriving-car-will-change-everything.asp)

Tesla (n.d.) Artificial Intelligence and Autopilot. [Online: tesla.com](tesla.com/AI)

Schroer A (16 Aug 2021) Artificial Intelligence in Cars Powers an AI Revolution in the Auto Industry [Online: builtin.com](https://builtin.com/artificial-intelligence/artificial-intelligence-automotive-industry)

Aten J (n.d.). Elon Musk's Ambitious Plan to Build a Self-Driving Tesla Has Just 1 Problem. It Doesn't Work. [Online: inc.com.](https://www.inc.com/jason-aten/elon-musks-ambitious-plan-to-build-a-self-driving-tesla-nhtsa-investigation.html)

Lambert F (20 Aug 2021). Tesla is actually going to make a 'Tesla Bot' humanoid robot for general purpose use. [Online: electrek.co.](https://electrek.co/2021/08/25/tesla-starts-hiring-roboticists-tesla-bot-humanoid-robot-project/)
[AI and AVs Refrences.docx](https://github.com/birkenkrahe/ai482/files/7102923/AI.and.AVs.Refrences.docx)

**Reply by birkenkrahe** [reply]
Thanks for the update, well done. References (esp. for a proposal) should ideally all be used in the text. Your next step is to turn this topic description into a research question - what exactly it is that you wish to explore. This will also narrow the concerns, and perhaps "test data" won't be an issue after all. Since there are so many open questions for the future of autonomous cars, it should be hard to find one. In fact, narrowing it down to a usable size may be more of an issue.

**Reply by SRhoden42** [reply]
How about something along the lines of, How can AI be applied to complex decision making for Autonomous Vehicles?

**Reply by birkenkrahe** [reply]
Still too large for such a short essay. But it's OK to run with this as a working title and then either specify using a subtitle or (even better) breaking the large question down in the introduction, i.e. what type of decisions? What type of AI?

**Reply by SRhoden42** [reply]
How about something along the lines of "The issues in decision making with EVs and Autonomous driving"?

**Reply by birkenkrahe** [reply]
I don't see how this is more specific than the previous version - in fact it's worse because now you have EVs AND AVs (I see no connection between them, if there is one, that might be an interesting topic). Suggestion: narrow it down on one specific issue and to one type of car. Since you seem to know something about EVs, pick them - more data available to you. Another route to a better research question: what does the literature say? In your systematic review, you only have AV articles. The first one (Kim et al 2020) and the 3rd (Cunneen et al 2019) seems to sample research questions - you could pick one of those. Btw, you were supposed to find 5 articles but you only listed 4, it seems.

**Reply by SRhoden42** [reply]
How about something along the lines of "how do we build artificial intelligence when we don’t even fully understand intelligence" in the same regards to how do we know reality is real? I know it's a bit of a 180, but I just feel like I will actually engage with this more over the course of the semester. I have an interest in EVs so I tried to apply it to the course without fully understanding the question and assignment at hand. I hope that this time I get closer to the "right" answer, if not I will try again :) 

Cheers!

**Reply by birkenkrahe** [reply]
I like it! To be sure: we're talking about your presentation (currently scheduled for Nov 10), right?

**Reply by SRhoden42** [reply]
Yes indeed, my solo presentation!

**Comment by lyonhakeem** [comment]
### Application of AI in Baseball

- I am passionate about the game of baseball and how modern day baseball uses data analytics acquired by using AI for player development
- A good move example of data-driven baseball would be Moneyball (Moneyball, 2011)
- I've been using radars, sensors, and other technologies to refine my game and I did not even realize that there are some applied to those technologies.
- One product that has some AI in it (that I also did not realize) is called hittrax (Hittrax,2011), where they use sensors to "track pitch location, point of impact, and exit speed by building a digital rendering of the real world. This digital simulation is then placed into real life, age appropriate fields to give you accurate feedback on ball flight and how a real life defense would react." (RBI Tricities)
- I haven't thought of a specific topic within AI in baseball yet, but I'm excited to research about it.
- Some topics I can branch into is Player Development or Professional Team Management (How to be more efficient in winning using data)

Some References:

RBI Tricities (n.d) Hittrax FAQ. [online: rbitricities.com](https://www.rbitricities.com/hittrax-faq)

HitTrax (2011). [online: hittrax.com ](hittrax.com/training)

Moneyball (September 9, 2011) Sony Pictures Releasing.

John Sturdivant (October 23, 2020) Building the AI-Driven Baseball Analytics Team. [online: datarobot.com](https://www.datarobot.com/blog/building-the-ai-driven-baseball-analytics-team/)

Carlos Garcia (September 24, 2020) How Professional Baseball is Using Artificial Intelligence. [online: itsgame7.com](https://itsgame7.com/how-professional-baseball-is-using-artificial-intelligence/)

Tom Kelly (n.d) What Major League Baseball Can Teach Us About AI in Business. [online: netsuite.com](https://www.netsuite.com/portal/business-benchmark-brainyard/industries/articles/cfo-central/artificial-intelligence-in-business.shtml)


[Systematic_Literature_Review_Example_Table.docx](https://github.com/birkenkrahe/ai482/files/7112928/Systematic_Literature_Review_Example_Table.docx)

**Reply by birkenkrahe** [reply]
Your inline references are not referenced at the end (or I don't see them), e.g. Moneyball, or Hittrax? As discussed in class (Sept 3), your systematic lit review is very good! Your next step is to turn the topic into research question, i.e. make it much smaller than the current title, and find something that you can actually research, validate, and present. Post it here as a reply.

**Reply by lyonhakeem** [reply]
A specific topic that I am thinking of is "The Impact of maximizing movement efficiency through AI to improve player development in Baseball"

**Reply by birkenkrahe** [reply]
Okay, then you have to back this specific topic up with a review of current literature. The more specific, current terms and concepts your topic contains, the easier it will be to find anything written anywhere on it. Or use pimary sources (a coach to talk to, or a player, or a developer of AI) - which is always faster and often better. I don't know anything about this domain (baseball, player development, movement efficiency), alas. You could also find AI that achieves something similar in another area (not necessarily sports) and look at the possibility of transferring it to baseball. E.g. I do know that movement efficiency is a big issue in robotics.

**Reply by birkenkrahe** [reply]
The book "Computers and Thought" from 1963 that I featured [in this diary post](https://github.com/birkenkrahe/ai482/blob/7fe53b046f8666373a1e71845bbb877bbfe80c16/diary.md#ai-books) contains an article "[Baseball: An Automatic Question Answerer](https://www.deepdyve.com/lp/association-for-computing-machinery/baseball-an-automatic-question-answerer-r0VLjiVYOZ)" (this is an agent system, later called 'expert system'). This might be interesting for you.

**Comment by MalcolmH870** [comment]
### How can AI improve the production in football (NFL)

- Technology is getting more advanced by the years and it is getting involved with everything including sports.
- I am not fully aware on how much technology is involved in sports in the health aspect but this would be the best way for me to learn.
- I wonder how much can adding artificial intelligence help the game of football for the players and the views because the number of 
  views are declining and the injuries are becoming more severe. 
- Health is becoming a big concern in sports and keeping people healthy so applying AI could be the next big thing.

**References**

 1. Miller, Wood, Crandall (2019) Using Artificial Intelligence to Advance Player Health and Safety 

2. Claudino, Daniel, Vieira de Souza, Serrao, Pereira, Nassis (2019) Current Approaches to the Use of Artificial Intelligence for Injury Risk Assessment and Performance Prediction in Team Sports: a Systematic Review  

3. Cohen, Andrew (2019) The NFL Will Use Amazon Web Services (AI and Machine Learning) to Study Injuries

4. Harpaz, Joe (2018) 3 Ways Artificial Intelligence Can Save The National Football League

5. West, Darrell (2021) How the NFL is using AI to evaluate players

**Websites**

1. Online: https://www.nfl.com/_amp/using-artificial-intelligence-to-advance-player-health-and-safety
2. Online: https://sportsmedicine-open.springeropen.com/articles/10.1186/s40798-019-0202-3
3. Online: https://www.sporttechie.com/nfl-amazon-web-services-injuries-player-safety/
4. Online: https://www.forbes.com/sites/joeharpaz/2018/01/11/3-ways-artificial-intelligence-can-save-the-national-football-league/?sh=405d9a597da7
5. Online: https://www.brookings.edu/blog/techtank/2021/03/18/how-the-nfl-is-using-ai-to-evaluate-players/

**Reply by birkenkrahe** [reply]
Please update your submission according to the class discussion: use inline references, make the references consistent (you seem to use different citation styles), and complete (place of publication is missing - Links are not places of publication). Check expressions: instead of saying colloquially "sports is a big thing", say why and how (and then provide a reference) - e.g. with regard to commerce. Be specific if possible (e.g. how much money is made in sports). Otherwise great setup for a project, and you even found a peer-reviewed article! The "Factoreal" reference is weak - I couldn't even see when it was published. Next step: decide if you want this as a team project or only as a solo presentation. Turn the topic into a research question (you need that for both the presentation and the project). As for most other participants, the topic is much too large still. Enjoy!

**Reply by birkenkrahe** [reply]
You're missing the systematic literature review. Please make sure to post it as an attachment to your submission!

**Comment by datoishere** [comment]
### AIMI Chapter 3.3 - Searching for Solutions

- This chapter goes wants us to go into problem-solving mode and makes the reader think about different algorithms to help us success at the task problem at hand effectively
- Using algorithms play a substantial part in computer science and I have no doubt that it have a big role in most operations that would make any type of AI program functional
- I should have a good (but basic) concepts of algorithms and how they should cleaning operate.
- One big concern I carry is that I have yet to fully understand the different types of algorithms as well as the most effective algorithms to use in different circumstances and how to incorporate them. However, this project should give me a good opportunity to help me learn and expand a fairly new concept that I have studied.

**References** 

Russell, S. J., Norvig, P. Chang, M.-wei. (2020) Artificial intelligence a modern approach. Pearson Education Limited. 

Erickson, J. (2019). Algorithms. Jeff Erickson. [https://jeffe.cs.illinois.edu/teaching/algorithms/book/Algorithms-JeffE.pdf](url)

Palo, N. D. (2017, August 14). Making a robot learn how to Move, Part 1 - evolutionary algorithms. [https://towardsdatascience.com/making-a-robot-learn-how-to-move-part-1-evolutionary-algorithms-340f239c9cd2 ](url)

**Reply by datoishere** [reply]
[Literature Review AI - Dat O.docx](https://github.com/birkenkrahe/ai482/files/7106227/Literature.Review.AI.-.Dat.O.docx)

**Reply by birkenkrahe** [reply]
Excellent literature review! Typo in first bullet point. When you say "I have no doubt", I immediately have doubts! Provide a reference for this claim! References are incomplete: place of publication is missing. Links are not relevant (though nice to have - behind the text). Next: narrow your topic for the presentation and wrap it in a research question. To use this in the AI application team project, you would have to identify an AI app that uses search (or more than one, shouldn't be too hard), or suggest issues, or suggest improvements (these last two are very hard - I would stick with the examples). But you can also just use this for the presentation. You (and we all) are going to learn a lot no matter what!

**Comment by AmberMcClurg** [comment]
[LiteratureReviewMcClurgFrerichs.docx](https://github.com/birkenkrahe/ai482/files/7102861/LiteratureReviewMcClurgFrerichs.docx)
### AI and VR

- I am interested in Virtual reality (VR) since I enjoy being inside a game and being able to look at the real world in my living room. 
- 	VR allows you to test drive cars, look at homes before buying, and lets you layout furniture before making a final choice. (Fade, 2019) Entertainment is a whole new level with VR when involving AI. For example, in a video game it can make the background characters be able to interact with real-life players smoothly. 
- What motivates me for this topic is learning more about how we are able to enter a virtual world using our bodies (Gillies, 2020) and how AI helps improve the experience.** 


**References** 

Fade, L. (2019, July 2). VR and AI: Two Technologies Set to Merge. VR Vision Group Online: vrvisiongroup.com

Gillies, M. (2020, May 8). Virtual Reality and Artificial Intelligence - Machine Learning for All. Medium. Online: medium.com

Panigrahi, D. (2021, April 7). Incorporating the Usage of 3D Artificial Intelligence With AR and VR Technology. Total Retail. Online: mytotalretail.com

**Reply by birkenkrahe** [reply]
Very cool topic, good recent references! Improvements: inline references should be shorter, e.g. "(Fade, 2019)" instead of the whole title. Nice project setup. Systematic lit review has good sources, but small mistakes in the full reference column (what is "1AD" in ref #4?) Place of publication missing for some. Links: here as on the web, hide them behind the text. In the reference, use only root URLs, e.g. "[Online: plato.stanford.edu](https://plato.stanford.edu/entries/turing-test) ". Missing in the literature review: any argument for the wholesale "medium" quality label in the column "credibility": articles on medium.com are not automatically "medium" quality (in fact, most are very low). Please update accordingly. Next step: identify research question(s) for presentation/project.

**Comment by jsampley** [comment]
### AI in the Military

- With the advent of human-piloted drones in warfare, people have likely become increasingly worried about the potential of lethal autonomous weapons

- Our military has recently begun employing autonomous drones in warfare, the first reported instance of such being just last year (Hernandez, 2021).

- While some might expect this technology to be too advanced or unregulated for manufacture in the present, this is simply not the case. Turkish company STM manufactures the KARGU drone, that which was utilized by the U.S. military last year. STM claims it has "multiple warhead options" and a "autonomous navigation and flight control system", capable of making "precision strike[s] with minimum collateral damage" ("KARGU", 2020).

- Concerned by a future increase in usage of these technologies in global warfare, the U.N. itself has opened a commission to investigate the subject and potentially institute a ban on autonomous lethal robots. When these plans were put in motion only five years ago, they claimed the technology had "not yet been developed" ("UN: Key Action on 'Killer Robots'", 2020).

### References

Hernandez, J. (2021, June 1). A Military Drone With a Mind of Its Own Was Used in Combat, U.N. Says. NPR. [Online: npr.org.](https://www.npr.org/2021/06/01/1002196245/a-u-n-report-suggests-libya-saw-the-first-battlefield-killing-by-an-autonomous-d)

KARGU - Rotary Attack Wing UAV. STM. (2020). [Online: stm.com.](https://www.stm.com.tr/en/kargu-autonomous-tactical-multi-rotor-attack-uav) 

UN: Key Action On 'Killer Robots'. Human Rights Watch. (2020, October 28). [Online: hrw.org.](https://www.hrw.org/news/2016/12/16/un-key-action-killer-robots#)

**Reply by jsampley** [reply]
[Sampley_Literature_Review.docx](https://github.com/birkenkrahe/ai482/files/7103546/Sampley_Literature_Review.docx)

**Reply by birkenkrahe** [reply]
Very well done, including the literature review. For the inline references, you don't need apostrophs and/or long author names - e.g. (UN, 2020) would be sufficient...if this was the author. In fact, "Human Rights Watch" is the author in this case, as far as I can see, so (HRW, 2020) would be fine. You used the title for the inline references - you can do that, of course, there are different styles, but then you have to do it always. And the inline ref should only be a pointer to the full ref. Otherwise - great topic, I'm dying (sorry for the pun) to learn more.

**Reply by birkenkrahe** [reply]
You now need to narrow this to a research question. Also, with "concerns", I did not mean concerns about the topic, but about your ability to do this on time and on quality.

**Comment by TokyoTurtle22** [comment]
Machine Learning AI in "Video" Games.

- I've always loved games and recently I've picked up playing online chess. As soon as I read it, I was immediately drawn to the topic idea of an AI that plays Checkers (Samuels, 1959). Which reminds me about the computer Deep Blue, which defeated chess grandmaster Garry Kasparov.
- Without machine learning we have to figure out exact ways of solving a problem in extreme detail, which can be time consuming and costly (Samuels, 1959).
- In Machine Learning it splits a 'population' into different outcomes, then determines which is fittest by comparing them to set goals. This relates to the computers "feelings". The computer can then learn from this 'population' (Francone, Nordin, Banzhaf, 1996).

Questions // Concerns
- What exactly is machine learning? How does the machine 'Learn'? I can code a machine to pick the optimal outcome based on data, but how would it 'learn' past experiences?
- What is a population?

**References**
Samuels (1959). Some Studies in Machine Learning Using the Game of Checkers.
Francone, Nordin, Banzhaf (1996). Computer implemented machine learning and control system.

**Reply by birkenkrahe** [reply]
I've only recently re-discovered my love for games, but I dig this topic! We're going to look at the Samuels checkers example in class (History of AI) - great stuff. The reference is missing the place of publication - pls update. Links to the articles might also be useful (though optional) - just nice for others to check them out quickly instead of having to search. Your concerns/questions are excellent. We'll look at this in class (I'm going to lecture on machine learning case studies soon). Some of these questions could lead to a glossary of terms, which is a good way to begin any research topic. Also: you have not added your literature review (using the template), please do. Any of your questions could become research questions - not in the sense that you exhaust them (they're too vast and vague) but that you present answers that you can find and understand so that your audience understands them. ML is only a part of AI.

**Comment by EmmaBrandes** [comment]
**How AI is used in soccer**

- I know a lot about soccer and am interested in how artificial intelligence can be used in helping athletes perform and helping them evolve into better players.
- I know that AI is used in helping athletes get better foot skills because I have a smart soccer ball. The ball has sensors inside that track your touches and It also has an app that you can download onto your phone that has videos and skills you can practice with. (Dribbleup) 
- AI can also be used to prevent injuries and provide real-time feedback and build personal training programs.(AI in Sports)  
- I’m not sure which direction I’m going to take my presentation yet. 

**References:**
https://dribbleup.com/products/smart-soccer-ball

https://www.day1tech.com/ai-in-sports-are-you-game-for-artificial-intelligence/#:~:text=Predictive%20analysis%20by%20AI%20can,strategies%2C%20and%20weaknesses%20during%20games 

Literature review: https://docs.google.com/document/d/1MjrgAocHzsSJb5PnWTVcLEFt7e2ftWM-pf1zSw-d-bo/edit

**Reply by birkenkrahe** [reply]
Hi Emma - great topic, and useful that some others also are interested in the AI vs. sports area of knowledge. So you have people to talk to (Malcolm, Hakeem). Your inline references need a year and you don't have references at all - links are not references. Please update. I cannot see your lit review yet (denied access) - as soon as I can, I'll give you feedback. You have a bunch of potential research questions here and you need to select and narrow them down. Finding a concrete app that utilizes AI to support soccer. SAP (the company) is a great player in this area by the way, and I know them well. They also have open courses on their applications. Check out SAP Leonardo (an analytics bundle). Please update according to my comments!

**Comment by fortuhf** [comment]
How AI is use in soccer by coaches to improve their teams and players.

- What motivates me to present this topic is my knowledge and love for soccer which for sure will make this easier.
- I have been playing and watching soccer since I was a kid and I have seen how the implementation of AI technology in this sport is helping the coaches to improve their players and also to prepare for games.
- AI technology in soccer helps coaches to know all the data about games.  Coaches with the help of AI technology like Instat (sports performance analysis company) can get to know better their teams and their opponents. Instat help them to see the stats that not everybody see in the games but that they are very important such as how many tackles a player won, pass accuracy, dribbles, lost balls and many other stats that will help coaches to see which players are being more effective in the field.
- This season we are using Instat and as a player I can say that it has been very helpful to have all my stats after every game with video of all my actions and the whole game. This is helping me to see in which areas I have to improve in order to be a better player and be able to help my team.

References:
https://instatsport.com/football/ricardo_pombo_interview

**Reply by birkenkrahe** [reply]
@fortuhf - here is a source I found on blockchain, if you're still interested: [a simple implementation using R.](https://www.datacamp.com/community/tutorials/blockchain-r).
